Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -194,15 +194,74 @@ class ConfigurationTests: XCTestCase {
     func testConfiguresCorrectlyFromDict() {
         let ruleConfiguration = [1, 2]
         let config = [RuleWithLevelsMock.description.identifier: ruleConfiguration]
-        let rules = testRuleList.configuredRules(with: config)
+        // swiftlint:disable:next force_try
+        let rules = try! testRuleList.configuredRules(with: config)
```

Comment:
I only found out about this when looking at the type signature from the `allTests` static member for XCTest Linux.